examples/basics/multi_output_example.exs

```diff
@@ -18,13 +18,13 @@ data = Stream.repeatedly(fn ->
 end)
 
 # Create the model
-input = Axon.input({nil, 1})
+model =
+  Axon.input({nil, 1})
+  |> Axon.dense(32, activation: :relu)
+  |> Axon.dense(64, activation: :relu)
 
-fc = Axon.dense(input, 32, activation: :relu)
-fc = Axon.dense(fc, 64, activation: :relu)
-
-out1 = Axon.dense(fc, 1)
-out2 = Axon.dense(fc, 1)
+out1 = Axon.dense(model, 1)
+out2 = Axon.dense(model, 1)

```

Comment:
The way this model was written was intentional to show that a multi-output model is actually just separate Axon objects in a container.

Reply:
Now I see what `c` in `fc` did stand for. :)
Isn't `model = {out1, out2}` line what represents the container?

Reply:
Yes, correct!

Reply:
Are you suggesting that current PR changes would conceal this aspect?

Reply:
Yes, I don't think it's necessary